src/Strategy.sol

```diff
@@ -323,7 +318,12 @@ contract Strategy is BaseStrategy {
         return _amtWithSlippage;
     }
 
+    function unstakeLPTokens(uint256 _amount) external onlyVaultManagers {
+        _unstakeLPTokens(_amount);
+    }
+
     function _unstakeLPTokens(uint256 _amount) internal {
+        require(_amount > 0, "Unstake abount must be greater than 0");
```

Comment:
change this to an `if` statement instead otherwise you'll get a lot of reverts and hinder normal operations